spark/pom.xml

```diff
@@ -292,7 +293,28 @@
       <version>${powermock.version}</version>
       <scope>test</scope>
     </dependency>
-
+<!--
+    <dependency>
+      <groupId>org.apache.mahout</groupId>
+      <artifactId>mahout-math</artifactId>
+      <version>${mahout.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.mahout</groupId>
+      <artifactId>mahout-math-scala_2.10</artifactId>
+      <version>${mahout.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.mahout</groupId>
+      <artifactId>mahout-spark_2.10</artifactId>
+      <version>${mahout.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.mahout</groupId>
+      <artifactId>mahout-spark-shell_2.10</artifactId>
+      <version>${mahout.version}</version>
+    </dependency>
+-->
```

Comment:
Ha! Commented them out to make sure it all worked. Forgot to delete before pushing...

Will fix now.